src/utils/retryAsking.js

```diff
@@ -0,0 +1,21 @@
+const readline = require("node:readline");
+const { stdin: input, stdout: output } = require("node:process");
+
+/**
+ * @param {string} query
+ * @param { (answer: string) => void } callback
+ */
+function retryAsking(query, callback) {
+  const rl = readline.createInterface({ input, output });
+  rl.question(query, (answer) => {
+    try {
+      rl.close();
+      callback(answer);
```

Comment:
@woowapark 의 설명을 기다리고 있습니다.

```suggestion
      callback(answer);
      rl.close();
```

Reply:
~~(글로 적다 포기한 흔적)~~     
오프라인에서 설명드리겠습니다!ㅋㅋ

Reply:
설명 완료!ㅋㅋ